service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!-- Generic Service Request -->
+    <service verb="set" noun="Connection">
```

Comment:
We can rename this to verb="send" noun="FedExRequest" to have a generic name and be consistent with the naming pattern using in all other integrations.